src/libs/actions/IOU.ts

```diff
@@ -1629,7 +1678,7 @@ function createSplitsAndOnyxData(participants, currentUserLogin, currentUserAcco
             optimisticPolicyRecentlyUsedCategories,
             optimisticPolicyRecentlyUsedTags,
             isNewOneOnOneChatReport,
-            shouldCreateNewOneOnOneIOUReport,
+            !!shouldCreateNewOneOnOneIOUReport,
```

Comment:
is `!!` needed? Isn't `shouldCreateNewOneOnOneIOUReport` boolean? If not - use `!!` everywhere (eg. 1579 line)